### DISCUSSION TITLE
Screen recorder

### DISCUSSION THREAD

**Discussion by douglasheld2** [elementary/triage#541] [Q&A]
I really love the screenshot app included in ElementaryOS 7.1. 
Today I need a screen recorder. To my surprise I am spending over an hour, trying numerous screen recorder apps that do not work.

It would be nice to have a screen recording app included and configured out of the box.

**Comment by douglasheld2** [comment]
In the end I think i've got this solved with building ffmpeg from source.

1. Configure the microphone as the input source and set the volume.
2. sudo apt install yasm libpusle-dev
3. ./configure --enable-libxcb --enable-libpulse
4. make
5. sudo make install
6. ffmpeg -probesize 10M -video_size 1200x800 -framerate 25 -f x11grab -i :0.0+0,0 -f pulse -ac 2 -i default  ~/screen.mp4

**Comment by archisman-panigrahi** [comment]
You can use simplescreenrecorder (available on the APT repositories). It does not work with Wayland, though

**Comment by leolost2605** [comment]
On OS 8 you should be able to use any screen recorder that makes use of the portal which should work both on X and on wayland. E.g. https://flathub.org/apps/io.github.seadve.Kooha is a pretty cool little recorder :)

**Comment by amiteeeblight** [comment]
I suppose that you could use OBS.

**Comment by davidak** [comment]
**GPU Screen Recorder** seem to be a very good option technically.

> This is a screen recorder that has minimal impact on system performance by recording your monitor using the GPU only, similar to shadowplay on windows. This is the fastest screen recording tool for Linux.

It works for me on NixOS with Pantheon Desktop.

The Flatpak version is probably the easiest way to install it for you: https://flathub.org/en/apps/com.dec05eba.gpu_screen_recorder
